tests/test_isrTaskLSST.py

```diff
@@ -134,17 +139,15 @@ def test_isrBootstrapBias(self):
         self.assertIn(key, metadata)
         self.assertEqual(metadata[key], "adu")
 
+        key = "LSST ISR READ NOISE UNITS"
+        self.assertIn(key, metadata)
+        self.assertEqual(metadata[key], "electron")
+
         for amp in self.detector:
             amp_name = amp.getName()
             key = f"LSST ISR GAIN {amp_name}"
             self.assertIn(key, metadata)
             self.assertEqual(metadata[key], isr_config.nominalGain)
-            key = f"LSST ISR READNOISE {amp_name}"
-            self.assertIn(key, metadata)
```

Comment:
Upon second thought, checking the existence of this keyword is a good idea so this part should be restored.

Reply:
I'm not sure what to do with the read noise units; they are not really electron in this case since we don't know the gain.  But don't change this part, I think when I change to the better "bootstrap" terminology on another branch it'll be clearer.